Detectors/MUON/MCH/Raw/Decoder/src/UserLogicElinkDecoder.h

```diff
@@ -186,9 +186,15 @@ bool UserLogicElinkDecoder<CHARGESUM>::append10(uint10_t data10)
         if (isSync(mSampaHeader.uint64())) {
           reset();
         } else if (mSampaHeader.packetType() == SampaPacketType::HeartBeat) {
-          sendHBPacket();
-          transition(State::WaitingHeader);
-          result = true;
+          if (mSampaHeader.isHeartbeat()) {
```

Comment:
this change is unrelated to this PR, isn't it ?

Reply:
Yes, indeed. I will remove the unrelated changes and prepare separate PR(s) for them.